diff --git a/ent/personal/schema/session.go b/ent/personal/schema/session.go
index c2e9f595..9d811a64 100644
--- a/ent/personal/schema/session.go
+++ b/ent/personal/schema/session.go
@@ -42,6 +42,12 @@ func (Session) Fields() []ent.Field {
 			Default(""),
 		field.String("workspace_dir").
 			Default(""),
+		field.String("provider_name").
+			Optional().
+			Nillable(),
+		field.String("model_name").
+			Optional().
+			Nillable(),
 		field.JSON("metadata", map[string]any{}).
 			Optional(),
 		field.Time("created_at").
diff --git a/ent/team/schema/session.go b/ent/team/schema/session.go
index 2b3fe8d2..7ea2d97e 100644
--- a/ent/team/schema/session.go
+++ b/ent/team/schema/session.go
@@ -43,6 +43,12 @@ func (Session) Fields() []ent.Field {
 		field.String("workspace_dir").
 			Optional().
 			Default(""),
+		field.String("provider_name").
+			Optional().
+			Nillable(),
+		field.String("model_name").
+			Optional().
+			Nillable(),
 		field.JSON("metadata", map[string]any{}).
 			Optional().
 			Default(map[string]any{}),
