diff --git a/ent/personal/schema/app.go b/ent/personal/schema/app.go
index 283c0e9d..36fdb08b 100644
--- a/ent/personal/schema/app.go
+++ b/ent/personal/schema/app.go
@@ -34,6 +34,8 @@ func (App) Fields() []ent.Field {
 			Default(1),
 		field.String("session_id").
 			Default(""),
+		field.String("provider_name").Optional().Nillable(),
+		field.String("model_name").Optional().Nillable(),
 		field.Time("created_at").
 			Default(time.Now).
 			Immutable().
diff --git a/ent/team/schema/app.go b/ent/team/schema/app.go
index 98895361..98e137eb 100644
--- a/ent/team/schema/app.go
+++ b/ent/team/schema/app.go
@@ -36,6 +36,8 @@ func (App) Fields() []ent.Field {
 		field.UUID("published_by", uuid.UUID{}).
 			Optional().
 			Nillable(),
+		field.String("provider_name").Optional().Nillable(),
+		field.String("model_name").Optional().Nillable(),
 		field.Time("created_at").
 			Default(time.Now).
 			Immutable().
